### PR TITLE
Be able to build multiple ngeo interfaces

### PR DESCRIPTION
**Pull request by sbrunner**
Also completely remove the Sencha touch interface from the create create template

Files per interfaces:
* `<package>/static-ngeo/less/<interface>.less`
* `<package>/static-ngeo/js/<interface>.js`
* `<package>/static-ngeo/js/<interface>controller.js`
* `<package>/template/<interface>.html`

Also fix: #1645

### CODE REVIEW

**Comment by sbrunner** [comment]
Shouldn't we merge the `<interface>.js` with the `<interface>controller.js`?

File: c2cgeoportal/scaffolds/create/apache/wsgi.conf.mako [hunk 1]
```diff
@@ -18,16 +18,26 @@ RewriteEngine on
 # http://code.google.com/p/modwsgi/wiki/FrequentlyAskedQuestions#Access_Control_Mechanisms
 WSGIPassAuthorization On
 
+
+% for interface in interfaces:
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/create/apache/wsgi.conf.mako]: Wouldn't it be clearer to use a single block of python here?
```
% for interface in interfaces:
<%
path = interface == "main" ? apache_entry_point :
    "%s%s/" % (apache_entry_point, interface)
%>
RewriteRule ^${path}?$ /${instanceid}/wsgi/${interface} [PT]
RewriteRule ^${path}theme/(.+)$ /${instanceid}/wsgi/${interface}/theme/$1 [PT]
% endfor
```

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 2]
```diff
@@ -4,21 +4,28 @@ This file includes migration steps for each release of c2cgeoportal.
 Version 2.0.0
 =============
 
+This changelog consider that you don't actually use an ngeo interface and you
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: consider => considers
actually => currently

**Comment by asaunier** [comment]
I have made a few comments, else no objection from my perspective.

**Pull request merged**